Makefile

```diff
@@ -32,6 +32,7 @@ lint:
 
 format:
 	black moto/ tests/
+	ruff --fix moto/ tests/
```

Comment:
When we perform `make lint`, we use `ruff`, but when formatting we use `black`. This creates a small inconsistency in the development workflow, especially because `ruff` mentions this:

```
❯ make lint
Running ruff...
ruff check moto tests
tests/test_sagemaker/test_sagemaker_model_package_groups.py:2:1: I001 [*] Import block is un-sorted or un-formatted
Found 1 error.
[*] 1 fixable with the `--fix` option.
make: *** [Makefile:23: lint] Error 1
```